layouts/org/single.html

```diff
@@ -0,0 +1,119 @@
+                        {{ if isset . "email"}}<a href="mailto:{{ .email }}"><img width="25" src="/about/email_icon.png"></a>{{ end }}
+                        <br>
+                        {{ end }}
+                    </p>
+                </div>
+            </li>
+            <li class="flex-item">
+                <h1>GR 4.0 Dev</h1>
+                <div style="text-align: left;">
+                <p style="line-height: 1.4;">
+                Lead:<br>
+                Josh Morman<br><br>
+                Members:<br>
+                Bastian Bloessl<br>
+                Marcus Müller<br>
+                </div>
+                </p>
+            </li>
+            <li class="flex-item">
+                <h1>GRC</h1>
+                <div style="text-align: left;">
+                    <p style="line-height: 1.4;">
+                    Lead:<br>
+                    Håkon Vågsether - grc@gnuradio.org<br><br>
+                    Members:<br>
+                    Derek Kozel<br>
+                    </div>
+                    </p>
+            </li>
+            <li class="flex-item">
+                <h1>Documentation</h1>
+                <div style="text-align: left;">
+                    <p style="line-height: 1.4;">
+                    Lead:<br>
+                    Marc Lichtman - docs@gnuradio.org<br><br>
+                    Members:<br>
+                    Barry Duggan<br>
+                    </div>
+                    </p>
+            </li>
+            <li class="flex-item">
+                <h1>GRCon</h1>
+                <div style="text-align: left;">
+                    <p style="line-height: 1.4;">
+                    Lead:<br>
+                    Samantha Palazzolo - grcon@gnuradio.org<br><br>
+                    Members:<br>
+                    Josh Morman<br>
+                    Marc Lichtman<br>
+                    Barry Duggan<br>
+                    Derek Kozel<br>
+                    </div>
+                    </p>
+            </li>
+            <li class="flex-item">
+                <h1 style="margin-bottom: 0px;">SigMF</h1>
+                <h4>(Signal Metadata Standard)</h4>
+                <div style="text-align: left;">
+                    <p style="line-height: 1.4;">
+                    Lead:<br>
+                    Jacob Gilbert - sigmf@gnuradio.org<br><br>
+                    Members:<br>
+                    Marc Lichtman<br>
+                    </div>
+                    </p>
+            </li>
+            <li class="flex-item">
+                <h1 style="margin-bottom: 0px;">VOLK</h1>
+                <h4>(SIMD Optimization)</h4>
+                <div style="text-align: left;">
+                    <p style="line-height: 1.4;">
+                    Lead:<br>
+                    Johannes Demel - volk@gnuradio.org<br><br>
+                    Members:<br>
```

Comment:
As per the other comment, we are "Co-Lead" here. Since we're adding this file, how about we get this corrected from the get-go?

Reply:
fixed!